src/lib/config.js

```diff
@@ -40,32 +40,29 @@ const ConvictConfig = Convict({
       default: 'reporting',
       env: 'EVENT_STORE_DB_EVENTS_COLLECTION',
     },
+    APPLY_SCHEMA: {
+      doc: 'Apply schema to existing collection',
+      format: Boolean,
+      default: false,
+      env: 'EVENT_STORE_APPLY_SCHEMA',
+    },
   },
   KAFKA: {
-    BROKER_LIST: {
-      doc: 'BROKER_LIST',
-      format: Array,
-      default: ['localhost:9092'],
-      env: 'KAFKA_BROKER_LIST',
-    },
-    CONSUMER_GROUP: {
-      doc: 'CONSUMER_GROUP',
-      format: String,
-      default: 'reporting_events_processor_consumer_group',
-      env: 'KAFKA_CONSUMER_GROUP',
-    },
-    CLIENT_ID: {
-      doc: 'CLIENT_ID',
-      format: String,
-      default: 'reporting_events_processor_consumer',
-      env: 'KAFKA_CLIENT_ID',
-    },
     TOPIC_EVENT: {
```

Comment:
I think this change in config is a breaking change. Please release major version.

Reply:
Yes, the commit contains BREAKING CHANGE:, so a major version should be released automatically.